src/main/kotlin/no/nav/helse/dokument/DokumentService.kt

```diff
@@ -18,19 +20,39 @@ private val dokumentContentTypeCounter = Counter.build()
 
 class DokumentService(
     private val dokumentGateway: DokumentGateway,
+    private val k9MellomlagringGateway: K9MellomlagringGateway,
     private val image2PDFConverter: Image2PDFConverter,
     private val contentTypeService: ContentTypeService
 ) {
 
-    suspend fun hentDokumenter(urls: List<URI>,
-                               aktoerId: AktoerId,
-                               correlationId: CorrelationId): List<Dokument> {
+    suspend fun hentDokumenter(
```

Comment:
Må ikke det som her heter `fødselsnummer` også være optional for å støtte både k9-dokument & k9-mellomlagring?

Også sjekke helt i starten at kun èn av dem er satt

Reply:
Samme grunn som https://github.com/navikt/k9-joark/pull/17#discussion_r509989112